docs/staging/freezes/request-a-freeze-exception.md

```diff
@@ -0,0 +1,162 @@
+(request-a-freeze-exception)=
+# Request a Freeze exception
+
+```{note}
+This page will be moved to:
+* contributors > advanced
+```
+
+Requests for {ref}`freeze-exceptions` are filed as bugs in Launchpad
+against the relevant package (or just "Ubuntu" if the package is not available
+yet).
+
+When you make your request, set the bug status to *New* to ensure the Release
+Team sees the request. Once all the required information is included, file your
+bug and subscribe [`ubuntu-release`](https://launchpad.net/~ubuntu-release) to
+it.
+
+
+## What to include
```

Comment:
as discussed you might make these two "what to include for small change vs large change" - that overcomes your confusion on those being almost the same